assembl/views/api2/synthesis.py

```diff
@@ -16,6 +20,30 @@ def discussion_notifications(request):
     return list(request.context._instance.get_notifications())
 
 
+@view_config(context=CollectionContext, renderer='json', request_method='GET',
+             ctx_named_collection="Discussion.syntheses", permission=P_READ,
+             accept="application/json")
+def get_syntheses(request, default_view='default'):
+    ctx = request.context
+    user_id = authenticated_userid(request) or Everyone
+    permissions = get_permissions(
+        user_id, ctx.get_discussion_id())
+    check_permissions(ctx, user_id, permissions, CrudPermissions.READ)
+    include_unpublished = P_EDIT_SYNTHESIS in permissions
+    view = request.GET.get('view', None) or ctx.get_default_view() or default_view
+    include_tombstones = asbool(request.GET.get('tombstones', False))
+    discussion = ctx.get_instance_of_class(Discussion)
```

Comment:
Why go through this deeply nested method? Go `Discussion.get(ctx.get_discussion_id())`.